packages/blade/src/components/Typography/Display/Display.tsx

```diff
@@ -18,6 +18,12 @@ export type DisplayProps = {
    */
   color?: BaseTextProps['color'];
   size?: Extract<BaseTextSizes, 'small' | 'medium' | 'large' | 'xlarge'>;
+  weight?: Extract<BaseTextProps['fontWeight'], 'regular' | 'medium' | 'semibold'>;
+  /**
+   *
+   * @default - 0% when weight is semibold, otherwise -1%
+   */
+  letterSpacing?: BaseTextProps['letterSpacing'];
```

Comment:
do we need to expose this to consumer? I thought we'll handle this internally always 🤔

Reply:
Aha right. We don't. Removed it https://razorpay.slack.com/archives/G01B3LQ9H0W/p1701170941397549